### PR TITLE
fetching parameters when user redirected from extension and showing p…

### PR DESCRIPTION
**Pull request by Meet5555**
fetching parameters when user redirected from extension and showing pricing popup based on parameters

1. free trial expired and basic plan
image

2. free trial expired and advance plan
image

3. plamexpired and basic plan
image

4. plan expired and advance plan
image

### CODE REVIEW

**Comment by Meet5555** [comment]
Popups after adding recommendation message

1. Free Trial Expired
 
image


2. Plan Expired
 
image

File: src/components/pricing/pricing.js [hunk 1]
```diff
@@ -34,18 +32,14 @@ export default function Pricing() {
 
     setUserCountry(country)
     if(lastPlan==='freeTrial'){
-      setUserFreeTrial(true)
-      setUserPlanExpired(false)
+      setUserLastPlan('freeTrial')
```
Comment (ShahJay147) [inline src/components/pricing/pricing.js:35]: setUserLastPlan(lastPlan)

File: src/components/pricing/pricing.js [hunk 2]
```diff
@@ -34,18 +32,14 @@ export default function Pricing() {
...
     }else if(lastPlan==='planExpired'){
-      setUserPlanExpired(true)
-      setUserFreeTrial(false)
+      setUserLastPlan('planExpired')
     }
     if(currentPlan==='basic'){
-      setIsUserBasic(true)
-      setIsUserAdvance(false)
+      setUserType('basic')
```
Comment (ShahJay147) [inline src/components/pricing/pricing.js:40]: setUserType(currPlan)

File: src/components/pricing/pricing.js [hunk 3]
```diff
@@ -19,11 +19,9 @@ export default function Pricing() {
   const [userPlanPeriod, setUserPlanPeriod] = useState('annually');
   const [planType, setPlanType] = useState('international');
   const [basicButton, setBasicButton] = useState('international');
-  const [userFreeTrial,setUserFreeTrial] = useState(false)
-  const [userPlanExpired,setUserPlanExpired] = useState(false)
+  const [userLastPlan,setUserLastPlan] = useState(null)
   const [userCountry,setUserCountry] = useState('')
-  const [isUserBasic,setIsUserBasic] = useState(false)
-  const [isUserAdvance,setIsUserAdvance] = useState(false)
+  const [userType,setUserType] = useState(null)
```
Comment (ShahJay147) [inline src/components/pricing/pricing.js:24]: userPlan

File: src/components/pricing/pricing.js [hunk 4]
```diff
@@ -217,190 +199,71 @@ export default function Pricing() {
     );
   }
 
-  return (
-    <div className="pricing">
-      {
-        userFreeTrial && isUserBasic && <>
+  function generatePricingPopup() {
+    return (
```
Comment (ShahJay147) [inline src/components/pricing/pricing.js:203]: let capPlan = userType.charAt(0).toUpperCase() + userType.slice(1)

**Pull request merged**